.travis.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         FILE_ENV="./ci/test/00_setup_env_native_tsan.sh"
 
     - stage: test
-      name: 'x86_64 Linux  [GOAL: install]  [bionic]  [no depends, only system libs, sanitizers: address/leak (ASan + LSan) + undefined (UBSan) + integer]'
+      name: 'x86_64 Linux  [GOAL: install]  [focal]  [no depends, only system libs, sanitizers: address/leak (ASan + LSan) + undefined (UBSan) + integer]'
```

Comment:
Maybe bump this Travis machine to focal in a separate PR?

Reply:
#19267